### PR TITLE
Add course page and material for Intro to ChIP-Seq

### LINKED ISSUE #67: To add course page and material for ChIP-Seq workshop
To add course page and pdf slides for the `Introduction to ChIP Sequencing and QC` workshop on December 11, 2024.

### PR DESCRIPTION
**Pull request by sharonfreshour**
# Purpose/implementation Section

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## What scientific question/module of analysis is your analysis addressing?

To add course page and slides for `Introduction to ChIP Sequencing and QC` workshop from December 11, 2024.

Adding files:
- README
- Introduction-to-ChIP-Sequencing-and-QC-2024-12-11.pdf

## What GitHub issue does your pull request address?

Closes #67 .

## Directions for reviewers 
Tell potential reviewers what kind of feedback you are soliciting.

### Which areas should receive a particularly close look?

- [x] Please check that appropriate folders and files have been added.
- [ ] Please make sure no file paths are visible in the slides.
- [ ] Please review README for clarity.
- [x] Please check that link to recording is accessible.

## Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.

### CODE REVIEW

File: courses/ChIP-Seq/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+| Audience | Computational Skills | Prerequisites | Duration |
+:----------|:----------|:----------|:----------|
+| Biologists/Bioinformaticians | Beginner | None | 45 min|
+
+
+## Access to the workshop recording
+
+St. Jude employees can access a [recording from the workshop](https://sjcrh-my.sharepoint.com/:v:/g/personal/sfreshou_stjude_org/EfAqM3KcgRlNho8TXNckL80Bl1JPTsb-k3F_t8W6c-Hewg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=ejIZLy).
```
Comment (AntoniaChroni) [inline courses/ChIP-Seq/README.md:22]:
```suggestion
St. Jude employees can access the [recording from the workshop](https://sjcrh-my.sharepoint.com/:v:/g/personal/sfreshou_stjude_org/EfAqM3KcgRlNho8TXNckL80Bl1JPTsb-k3F_t8W6c-Hewg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=ejIZLy).
```

File: courses/ChIP-Seq/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+## Below is the main directory structure listing the analyses and data files used in this repository
+
+```
+├── lectures
+|   ├── Introduction-to-ChIP-Sequencing-and-QC-2024-12-11.pdf
```
Comment (AntoniaChroni) [inline courses/ChIP-Seq/README.md:29]:
```suggestion
|   └── Introduction-to-ChIP-Sequencing-and-QC-2024-12-11.pdf
```

File: courses/ChIP-Seq/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## Course Authors
+
+Sharon Freshour, PhD ([@sharonfreshour](https://github.com/sharonfreshour))
+
+
+## Contact
+
+Contributions, issues, and feature requests are welcome! Please feel free to check [issues](https://github.com/stjudeDNBBinfCore/Trainings/issues).
```
Comment (AntoniaChroni) [inline courses/ChIP-Seq/README.md:40]:
```suggestion
Contributions, issues, and feature requests are welcome! Please feel free to check [issues](https://github.com/stjude-dnb-binfcore/trainings/issues).
```

**Review by AntoniaChroni — COMMENTED** [review]
Congrats on your first feature branch and PR for the core, great job! 🎉

Some suggestions for the current and future ones:

- Make sure you assign yourself the issue and PR and the right label for organization purposes and easy access to the archive.
- Add a list of elements to review on the ticket issue and/or PR along with the added files. It was unclear the description on the issue. 
- If you don't need something from the `Type of change` list on the PR, you can remove and keep/check the one that describes the PR's purposes (for less clutter and more clarity).
- Links work as expected.
- `README` file: There are minor suggestions.
- Slides files:
   1. In cases where you have: `Adapted from doi...`, you should add the link to the url so the user can simply click on it and easily access the source.
   2. Slide 27: I think that there are still some sample ID shown under the `Coverage` folder.
   3. Slides 28-29 (or anywhere else that this is the case): Remember to add all links to the relevant part of the phrase to reduce clutter on the slide and make it more accessible.
   4. Slide 29: To rewrite `We’ll walk through html example` to `We will walk through html example.` . It should be a complete sentence. 
   5. Slide 29: To rewrite `Can download html to follow along here:` to `You can download the HTML to follow along here:`. 
   6. Slide 29: When creating a list on a slide, it's important to choose one consistent style—whether complete sentences or just keywords—and stick with it throughout for clarity and uniformity.

**Comment by sharonfreshour** [comment]
>   2. Slide 27: I think that there are still some sample ID shown under the `Coverage` folder.

The file names that are visible on Slide 27 show the information related to ChIP-seq marks and proteins, nothing that should be able to identify the samples, so I think it is okay to leave them visible and I feel it is informative for explaining the folder/file structure.

**Comment by sharonfreshour** [comment]
Slides have been updated per suggestions here [10c68a8](https://github.com/stjude-dnb-binfcore/trainings/pull/68/commits/10c68a889b14e18c50c07accf52dc28843bc92ea)

**Review by AntoniaChroni — APPROVED** [review]
That looks great, I will go ahead and approve, great job!

One last suggestion about references on a slide is to follow the same format across slides. A good example is as you did for slide 5, i.e., `Adapted from The ENCODE Project Consortium (2011). PLOS Biology`.  But for, e.g., slide 9, you did `Adapted from doi.org/10.1038/nrg2641`. I would suggest for these cases to do instead `Adapted from Park (2009). Nature Reviews Genetics.` (with inserting the hyperlink on the citation part).

**Comment by AntoniaChroni** [comment]
> That looks great, I will go ahead and approve, great job! 🙌
> 
> One last suggestion about references on a slide is to follow the same format across slides. A good example is as you did for slide 5, i.e., `Adapted from The ENCODE Project Consortium (2011). PLOS Biology`. But for, e.g., slide 9, you did `Adapted from doi.org/10.1038/nrg2641`. I would suggest for these cases to do instead `Adapted from Park (2009). Nature Reviews Genetics.` (with inserting the hyperlink on the citation part).

@sharonfreshour You missed `.` at the end of all citations. For example: `Adapted from Wu (2015). Frontiers in Genetics` instead of `Adapted from Wu (2015). Frontiers in Genetics.`

**Pull request merged**